SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1456 @@
+	#addressing comparison reports
+	if ($doComparison -eq $true) {
+		foreach($eachNode in $ppmEnvironmentalDetails){
+			$eachNode.productAttributes.driftAlarm = "Green"
+			$eachNode.productAttributes.driftMessage = ""
+			foreach ($eachAttribute in $eachNode.productAttributes.PSObject.Properties) {
+				foreach ($product in $ppmStandardConfigValues) {
+					if ($product.productType -eq $eachNode.productType) {
+						if ($eachAttribute.Value -ne $null) {
+							if (($eachAttribute.Value -ne $product.productAttributes.$($eachAttribute.Name)) -and ($eachAttribute.Name -ne "driftAlarm") -and ($eachAttribute.Name -ne "driftMessage") -and ($eachAttribute.Name -ne "transportNodeID")) {
+								$eachNode.productAttributes.driftAlarm = "Red"
+								if ($eachNode.productAttributes.driftMessage -eq "") {
+									$eachNode.productAttributes.driftMessage = $eachAttribute.Name + "("+ $product.productAttributes.$($eachAttribute.Name) + ")"
+								}
+								else {
+									$eachNode.productAttributes.driftMessage = $eachNode.productAttributes.driftMessage + ", " + $eachAttribute.Name + "("+ $product.productAttributes.$($eachAttribute.Name) + ")"
+								}
+								$count += 1
+							}
+						} else {
+							$eachAttribute.Value = "NULL"
+							$eachNode.productAttributes.driftAlarm = "Red"
+							if ($eachNode.productAttributes.driftMessage -eq "") {
+								$eachNode.productAttributes.driftMessage = $eachAttribute.Name + "("+ $product.productAttributes.$($eachAttribute.Name) + ")"
+							} else {
+								$eachNode.productAttributes.driftMessage = $eachNode.productAttributes.driftMessage + ", " + $eachAttribute.Name + "("+ $product.productAttributes.$($eachAttribute.Name) + ")"
+							}
+							$count += 1
+						}
+					}
+				}
+			}
+		}
+		if ($onlyDrift -eq $true) {
+			$ppmEnvironmentalDetails | where { $_.productAttributes.driftAlarm -match "Red" } | ConvertTo-Json -depth 100 | Out-File $ppmVariables.outputFilePath
+		} else {
+			$ppmEnvironmentalDetails | ConvertTo-Json -depth 100 | Out-File $ppmVariables.outputFilePath
+		}
+	} else {
+		$ppmEnvironmentalDetails | ConvertTo-Json -depth 100 | Out-File $ppmVariables.outputFilePath
+	}
+	Write-LogMessage -type INFO -Message "$($ppmVariables.outputFilePath) report generated" -Colour Yellow
+}
+
+################################################################################
+####   configure environment password policies settings from the 			####
+####   standard configuration file 											####
+################################################################################
+Function SetEnvironmentPasswordPolicy {
+	Param (
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [psobject]$ppmVariables,
+		[Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmStandardConfigValues,
+        [Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmEnvironmentalDetails
+	)
```

Comment:
Add line space after Param block

Reply:
addressed in 9ee41934dd0ee5a302dd128339b72e2f82898175